redis/scripts/setup-drupal-settings.sh

```diff
@@ -4,11 +4,11 @@ set -e
 
 if [[ $DDEV_PROJECT_TYPE != drupal* ]] || [[ $DDEV_PROJECT_TYPE =~ ^drupal(6|7)$ ]] ;
 then
-  exit 0
+  exit 1
```

Comment:
Why does this have to be called out as an error?

Reply:
Because I remove the files only if this script fails:

```
if ! redis/scripts/setup-drupal-settings.sh; then
  # remove files
fi
```

That's why I return an error here.
Otherwise I would need to duplicate Drupal checks in `install.yaml`

Reply:
How about checking to see if it's drupal and if not removing files? Rather than depending on the script? Or is that the same?

Reply:
If I remove the file from the file itself, will this work for all environments?

I thought it could have unpredictable results, for example, for Windows.

Reply:
Okay, I'll try to remove the files here and see if it works.

Reply:
It works for me.